mass: 0.302

inertia:
  Ixx: 1.837e-3
  Iyy: 1.837e-3
  Izz: 2.7786e-4

offset:
  upper: 0.165
  lower: 0.103

linkage_factor:
  upper: 0.659294
  lower: 0.631363

spring_constant:
  upper: 0.074306
  lower: 0.190327

thrust_factor:
  upper: 2.03195e-5
  lower: 3.69007e-5

moment_factor:
  upper: 1.0169e-6
  lower: 8.94525e-7

following_time:
  bar: 1.78956
  motors:
    upper: 0.192567
    lower: 0.115429

speed_conversion:
  slope:
    upper: 421.3723
    lower: 437.7036
  offset:
    upper: -0.2555
    lower: -2.9047

phase_lag:
  slope:
    upper: -0.00206728
    lower: 0.00181741
  offset:
    upper: 0.176401
    lower: 0.0145523

max_swashplate_angle: 0.26